ngraph_creator/operations/src/Reshape.cpp

```diff
@@ -29,9 +29,42 @@ std::shared_ptr<ngraph::Node> Reshape::createNode() {
     const auto& dimsOperandIndex = sModelInfo->getOperationInput(mNnapiOperationIndex, 1);
     auto outDims = sModelInfo->GetConstVecOperand<int32_t>(dimsOperandIndex);
     VLOGDIMS(L3, outDims, "Reshape::createNode dims");
+
     auto inputIndex = sModelInfo->getOperationInput(mNnapiOperationIndex, 0);
     auto inputOp = mNgraphNodes->getOperationOutput(inputIndex);
-    ALOGV("%s outDims.size=%d", __func__, outDims.size());
+    const auto& inDims = getInputOperandDimensions(0);
+    auto numInputElements = 1;
+    int strechDim = -1;
+    auto numOutputElements = 1;
+
+
+    for (auto i = 0; i < inDims.size(); i++)
+        numInputElements*=inDims[i];
+
+    ALOGV("%s outDims.size=%d inDims.size=%d", __func__, outDims.size(),inDims.size());
+
+    for (auto i = 0; i < outDims.size(); i++) {
+        ALOGD("operand1: shape of output tensor outDims[%d] = %d ", i, outDims[i]);
+        if ((int)outDims[i] < 0) {
+            strechDim = i;
+            ALOGD("strechDim = %d", i);
```

Comment:
Use ALOGD, only if you expect the log to be printed in every execution of this code.
Can you add the __func__ to all the logs, so that it'll be helpful in debugging.